src/index.html

```diff
@@ -2,21 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Stop watch</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
According to the BEM methodology, modifiers should be used with the block or element class name. So it should be `class="stopwatch stopwatch--speed-up"`, not just `class="stopwatch--speed-up"`. Ensure that the base class is also included whenever a modifier class is used.